geomstats/_backend/pytorch/__init__.py

```diff
@@ -331,20 +350,25 @@ def einsum(*args, **kwargs):
     einsum_str = args[0]
     input_tensors_list = args[1:]
 
+    if len(input_tensors_list) == 1:
+        return torch.einsum(einsum_str, input_tensors_list)
+
     einsum_list = einsum_str.split('->')
     input_str = einsum_list[0]
-    output_str = einsum_list[1]
+    if len(einsum_list) > 1:
+        output_str = einsum_list[1]
```

Comment:
great, but won't line 411 crash if `output_str` is not defined?

Reply:
True. I have added the condition there as well.